react-state-3/react-state-3_tags/src/App.jsx

```diff
@@ -6,10 +6,22 @@ import "./App.css";
 export default function App() {
   const [tags, setTags] = useState(["JavaScript", "React", "CSS", "HTML"]);
 
+  function handleAddTag(newTag) {
+    setTags([...tags, newTag]);
+    console.log()
+    console.log("Add", newTag)
+  }
+
+  function handleDeleteTag( onDeleteTag ) {
```

Comment:
I would use a different paramenter name here and not `onDeleteTag`. Because by convention onDeleteTag is how we name the event fired. Here this is more `tagName`, or `tagNameToDelete`